lib/src/linden/xcolors.dart

```diff
@@ -40,6 +40,8 @@ class XColors {
   static const _orange = Color(0xffF29100);
   static const _white = Color(0xffffffff);
   static const _beige = Color(0xfff8f0e3);
+  static const _pink20 = Color(0xffF3CDDE);
+  static const _pinkNew = Color(0xffDB1870);
```

Comment:
We should probably use names from the Design System.

Reply:
yeah, it's called `_pink` in the new design system, and we already have a `_pink` here, I named it `_pinkNew` 🤷‍♂️

Reply:
Then update the old color to use the new hex value. I will be working on updating the colors soon anyway.

Reply:
didn't know it's ok to do so.. will do now.

Reply:
[a060c10](https://github.com/xaynetwork/xayn_design/pull/69/commits/a060c10fbff75619a61ccb7ac0e144e1461acc70)